examples/rsp-next-ts/styles/globals.css

```diff
@@ -0,0 +1,16 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
```

Comment:
Do we need to do any font-family loading of the adobe fonts?

Reply:
I think some people might use this as a template for their app, so probably not.